### PR TITLE
chore: add more ui components, ref leather-wallet/issues#64

### PR DESCRIPTION
**Pull request by pete-watters**
This PR adds another batch of UI components to the mono-repo. 

Issues I had hit with react errors were solved by using path aliases in the extension's `webpack.config.base.js`:
```

const aliases = {
...
  'react': path.resolve('./node_modules/react'),
  'react-dom': path.resolve('./node_modules/react-dom')
```

I'd like to get this merged so we can release a new package and I can work on consuming more components in the extension 

## Summary by CodeRabbit

- **New Features**
  - Introduced a variety of new UI components including Accordion, AddressDisplayer, DropdownMenu, IconButton, Input, Select, Tabs, Toast, and Tooltip.
  - Added Storybook stories for new components to demonstrate their usage and configurations.
  - Enhanced accessibility with components like AccessibleIcon.
  - Added utility functions for improved component functionality and styling.

- **Refactor**
  - Reorganized exports to streamline component access and usage.

- **Chores**
  - Updated `package.json` to include new dependencies from the `@radix-ui` library.

### CODE REVIEW

File: packages/ui/src/components/dropdown-menu/dropdown-menu.web.tsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  alignItems: 'center',
+  '--base-menu-padding': '0px',
+  bg: 'ink.background-primary',
+  borderRadius: 'xs',
+  boxShadow:
+    '0px 12px 24px 0px rgba(18, 16, 15, 0.08), 0px 4px 8px 0px rgba(18, 16, 15, 0.08), 0px 0px 2px 0px rgba(18, 16, 15, 0.08)',
+  p: '0',
+  willChange: 'transform, opacity',
+  zIndex: 999,
+  // TODO re-enable these and check if Settings tests still fail
```
Comment (pete-watters) [inline packages/ui/src/components/dropdown-menu/dropdown-menu.web.tsx:68]: This animation is still [causing extension shard 3 to fail](https://github.com/leather-wallet/extension/pull/5518) with timeouts in the `add-network.spec.ts`

File: packages/ui/src/components/input/input.web.tsx [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+  useState,
+} from 'react';
+
+import { sva } from 'leather-styles/css';
+import { SystemStyleObject } from 'leather-styles/types';
+
+import { propIfDefined } from '@leather-wallet/utils';
+
+import { createStyleContext } from '../../utils/style-context.web';
+//FIXME leather-wallet/issues#64: This is a double up of a hook that already exists in the codebase.
```
Comment (pete-watters) [inline packages/ui/src/components/input/input.web.tsx:18]: We are only using this `useOnMount` hook once in `input`. When removing the `input` from the extension I will investigate the best place to share it.

File: packages/ui/src/components/input/use-on-mount.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { useEffect } from 'react';
+
+import { isFunction } from '@leather-wallet/utils';
+
+/**
+ * @deprecated
```
Comment (pete-watters) [inline packages/ui/src/components/input/use-on-mount.ts:6]: See [comment here](https://github.com/leather-wallet/mono/pull/201/files#r1633088633)

**Review by fbwoolf — APPROVED** [review]

File: packages/ui/src/components/dropdown-menu/dropdown-menu.web.tsx (hunk 1 above)
Comment (pete-watters) [reply packages/ui/src/components/dropdown-menu/dropdown-menu.web.tsx:68]: We have this task https://github.com/leather-wallet/extension/issues/5054 

I should have added it in the comment, I'll add it

**Pull request merged**